pkg/services/provisioning/values/values_test.go

```diff
@@ -0,0 +1,176 @@
+package values
+
+import (
+	. "github.com/smartystreets/goconvey/convey"
+	"gopkg.in/yaml.v2"
+	"testing"
+)
+
+func TestValues(t *testing.T) {
```

Comment:
I think this kind of tests is better written in normal golang tests instead of convey. since convey requires quite some ceremony. Its briefly mentioned in the style guide docs but It's fairly new so most part of the codebase still uses convey.

Reply:
At first I would say I am all for but while trying to change it, I started to see the appeal of Convey and now am hesitant to change this :D. 2 issues with the golang tests I have:

1. They would be a bit more verbose as I do not see any way to do BeforeEach kind of setup in one place, just running a setup code/function at the start of each test. This is not a big deal and probably subjective as Convey can seem a bit magical but still kinda nice.
2. Better asserts which I think is more substantial. I assume there are some standalone assert libraries we could use. Here my main issue is with JSONValue test where I compare nested structures. Convey gives you nice visual diffs if they do not match and without it I would probably spend much more time debugging that. Without Convey, only thing I can think of is `reflect.DeepEqual` which only gives you `bool` response.

Reply:
Let's keep it this way and bring it up at the next backend call.